frontend/packages/topology/src/components/graph-view/Topology.tsx

```diff
@@ -177,7 +177,16 @@ const Topology: React.FC<TopologyProps &
     newVisualization.addEventListener(GRAPH_POSITION_CHANGE_EVENT, onCurrentGraphModelChange);
 
     if (storedLayout) {
-      graphModel.graph.layout = storedLayout.layout;
+      // Cleanup deprecated ColaForce layout. Otherwise the newVisualization.fromModel
+      // call will crash in @patternfly/react-topology Visualization `getLayout(type: string)`
+      if (storedLayout.layout === DEPRECATED_COLAFORCE_LAYOUT) {
```

Comment:
We only support the one layout now, why not just force it regardless of what the value is? It would also crash if it were anything else, right?

Reply:
That's true. There is some more code which stores the Layout and I don't want remove it completely. I just though this would be the easiest solution if we add another in the future again.

But I can also add list of supported layouts (currently just one) and set it to the default if it's unknown?!

Reply:
Updated, wdyt?